src/constants/bucketNames.ts

```diff
@@ -0,0 +1,6 @@
+export const MEASURES = 'measures';
```

Comment:
Wondering if would be better to put here a short comment that this is too gooddata AD specific which means it doesn't belong to `gooddata/typings`.

Reply:
I am not sure if comment is needed. All bucket naming is connected with GDC visualizations. Nothing generic to be moved to typings